### PR TITLE
fix(storefront): BCTHEME-269 Remove layout shifts to improve CLS indicator from Lighthouse performance report

### PR DESCRIPTION
**Pull request by yurytut1993**
#### What?

This PR has update for performance by removing layout shift caused by main navigation - after fonts loading number of strings of the navigation can be changed. 
Here is also performance comparison before fix and after (see screenshots).

#### Tickets / Documentation

[Ticket](https://jira.bigcommerce.com/browse/BCTHEME-269)

#### Screenshots (if appropriate)

after fonts loaded
Screenshot 2020-10-09 at 18 10 15

while fonts loading
Screenshot 2020-10-09 at 18 03 11

before
Screenshot 2020-10-09 at 17 26 55

after
Screenshot 2020-10-09 at 17 28 17

### CODE REVIEW

File: templates/layout/base.html [hunk 1]
```diff
@@ -25,6 +25,16 @@
         </script>
         <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js'}}"></script>
 
+        <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js"></script>
+
+        <script>
+            WebFont.load({
+                custom: {
+                families: ['Karla', 'Roboto', 'Source Sans Pro']
```
Comment (BC-tymurbiedukhin) [inline templates/layout/base.html:33]: Could you please explain how you defined list of the fonts?

Comment (yurytut1993) [reply templates/layout/base.html:33]: This fonts are variants we have to setup
Screenshot 2020-10-12 at 11 37 38

Comment (BC-tymurbiedukhin) [reply templates/layout/base.html:33]: 1. Would be nice if we could find a way how to load only relevant font
2. While loading page Monsterrat font is fetching
Screenshot 2020-10-12 at 14 54 32
3. From theme setting we can see Arial font at least
Screenshot 2020-10-12 at 14 56 59

Comment (BC-tymurbiedukhin) [reply templates/layout/base.html:33]: How many requests do we send for fetching fonts after your changes?

Comment (yurytut1993) [reply templates/layout/base.html:33]: > How many requests do we send for fetching fonts after your changes?

3 font requests - as it was

**Comment by BC-tymurbiedukhin** [comment]
LGTM, but don't forget to add changelog item, thank you

**Review by bc-alexsaiannyi — APPROVED** [review]

**Comment by yurytut1993** [comment]
> LGTM, but don't forget to add changelog item, thank you

done

**Review by BC-tymurbiedukhin — APPROVED** [review]

**Pull request merged**